server.js

```diff
@@ -1,10 +1,16 @@
-var Server = require('node-static').Server,
-    http = require('http'),
-    util = require('util');
+var Server        = require('node-static').Server;
+var http          = require('http');
+var createChannel = require('./server/channel.js').createChannel;
+var onConnection  = require("./server/fileLoader.js").onConnection;
 
-var port = 8080;
-var fileServer = new(Server)('.');
-var server = http.createServer(handleRequest);
+var port          = 8080;
+var fileServer    = new(Server)('.');
```

Comment:
I have no idea, I didn't write that, I left it that way assumed you had a reason for it :p